src/promptflow/promptflow/exceptions.py

```diff
@@ -209,3 +219,101 @@ class ValidationException(UserErrorException):
     """Exception raised when validation fails."""
 
     pass
+
+
+class _ErrorInfo:
```

Comment:
more unit test for this class

Reply:
especially for some key logic such as reversed error messages, status code filter, etc.

Reply:
More tests have been added.